init.sql

```diff
@@ -4,3 +4,8 @@ CREATE TABLE IF NOT EXISTS `organization` (
     Id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
     name VARCHAR(100)
 ) ENGINE=InnoDB;
+CREATE TABLE IF NOT EXISTS `user` (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    emailId VARCHAR(100),
+    password VARCHAR (20)
```

Comment:
let's use `CHAR(32)`. 
Because we do not store passwords directly. (if we do, anyone has access to the DB can steal passwords) We need to hash them. A popular solution is to use md5. Md5 always produce a string of 32 characters.

Read about md5.

Reply:
Fixed with bfe9979 @jayasanka-sack

Reply:
not the email. Password

Reply:
Fixed with https://github.com/pasindur99/Organizations-backend/pull/24/commits/68376222487bca0076b924142999c7fb5a685978